shell/src/test/java/org/apache/accumulo/shell/commands/QuotedStringTokenizerTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.shell.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class QuotedStringTokenizerTest {
+
+  @Test
+  public void testProperties() {
+    String jsonProperty = "config -s tserver.compaction.major.service.meta.planner.opts.executors="
+        + "[{\"name\":\"small\",\"type\":\"internal\",\"maxSize\":\"32M\",\"numThreads\":2},"
+        + "{\"name\":\"huge\",\"type\":\"internal\",\"numThreads\":2}]";
+    testTokens(jsonProperty, 3);
```

Comment:
Before this change needed to wrap the entire key value in an outer single quote to make it work.  Would be nice to test that still works.

```suggestion
    String jsonProperty = "config -s tserver.compaction.major.service.meta.planner.opts.executors="
        + "[{\"name\":\"small\",\"type\":\"internal\",\"maxSize\":\"32M\",\"numThreads\":2},"
        + "{\"name\":\"huge\",\"type\":\"internal\",\"numThreads\":2}]";
    testTokens(jsonProperty, 3);
    
     jsonProperty = "config -s 'tserver.compaction.major.service.meta.planner.opts.executors="
        + "[{\"name\":\"small\",\"type\":\"internal\",\"maxSize\":\"32M\",\"numThreads\":2},"
        + "{\"name\":\"huge\",\"type\":\"internal\",\"numThreads\":2}]'";
      testTokens(jsonProperty, 3);
```